content/jobs.njk

```diff
@@ -0,0 +1,59 @@
+---
+title: Jobs
+---
```

Comment:
You can add a `description` property here if you want that will be picked up by social embeds.  Without a description, social embeds just fall back to the default "One framework for creating powerful cross-platform games."  

The description would be shown here on social embeds. Completely optional though, if none provided it just defaults.

![image](https://github.com/MonoGame/monogame.github.io/assets/103014489/d912b17a-1701-43d6-b295-e210a690fe84)